### ISSUE TITLE
Upgrade to Angular 10

### ISSUE THREAD

**Issue by achapman009** [angular-material-extensions/password-strength#302]
### Bug Report or Feature Request (mark with an `x`)
```
- [x ] bug report -> please search issues before submitting
- [ ] feature request
```

### OS and Version?
 Windows 10

### Versions
 6.0.0

### Repro steps

ng update @angular/cdk

### The log given by the failure

Package "@angular-material-extensions/password-strength" has an incompatible peer 
dependency to "@angular/cdk" (requires "^9.0.0", would install "10.0.0").

### Desired functionality
 Upgrade to Angular 10 dependencies.

**Comment by YZahringer** [comment]
Same issue
````
npm WARN @angular-material-extensions/password-strength@6.0.0 requires a peer of @angular/common@^9.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @angular-material-extensions/password-strength@6.0.0 requires a peer of @angular/core@^9.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @angular-material-extensions/password-strength@6.0.0 requires a peer of @angular/forms@^9.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @angular-material-extensions/password-strength@6.0.0 requires a peer of @angular/cdk@^9.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @angular-material-extensions/password-strength@6.0.0 requires a peer of @angular/material@^9.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @angular-material-extensions/password-strength@6.0.0 requires a peer of tslib@^1.10.0 but none is installed. You must install peer dependencies yourself.
````

**Comment by yerevin** [comment]
#305

**Comment by AnthonyNahas** [comment]
I've just published a new version of the library [v7.0.0](https://www.npmjs.com/package/@angular-material-extensions/password-strength)


thank you 🍻
